admin-dev/themes/new-theme/scss/pages/_modules.scss

```diff
@@ -611,30 +683,51 @@ form#importDropzone {
     text-align: center;
     vertical-align: middle;
     img {
-      max-width: 57px;
+      max-width: 40px;
     }
   }
 
-  .module-column-infos {
+  .module-column-infos,
+  .module-column-description {
     display: table-cell;
     height: 50px;
     vertical-align: middle;
+    font-size:.95em;
     .module-name {
-      font-size: 1.2em;
+      color: #25b9d7;
     }
 
     .module-description {
-      @extend .text-muted;
+      color: #363a41;
     }
   }
 
+  .module-column-infos {
+    width: 200px;
+    padding: 0 4px;
+  }
+
+  .module-column-description {
+    width: 679px;
+  }
+
+  .module-column-description.hasColumnPosition {
```

Comment:
This can go inside the previous block:

```scss
  .module-column-description {
    width: 679px;
    
    &.hasColumnPosition {
      width: 600px;
    }
  }

Reply:
:ok_hand: